sunpy/map/mapsequence.py

```diff
@@ -359,21 +360,18 @@ def annotate_frame(i):
         def updatefig(i, im, annotate, ani_data, removes):
             while removes:
                 removes.pop(0).remove()
-
             im.set_array(ani_data[i].data)
-            im.set_cmap(kwargs.get('cmap', ani_data[i].plot_settings['cmap']))
-            norm = deepcopy(kwargs.get('norm', ani_data[i].plot_settings['norm']))
-
+            im.set_cmap(kwargs.get('cmap', ani_data[i].plot_settings.get('cmap')) or "grey")
+            norm = deepcopy(kwargs.get('norm', ani_data[i].plot_settings.get('norm')))
             if clip_interval is not None:
                 vmin, vmax = _clip_interval(ani_data[i].data, clip_interval)
+                if norm is None:
+                    norm = ImageNormalize()
```

Comment:
Not sure if this is the correct one to set but if its missing and you pass in a clip_interval, it goes boom.

Reply:
Great question, I can add a test.

Reply:
I added something that looks like a test, probs wrong